Description: Remove duplicate code
 Upstream code contains duplicate code (requests and urllib3).
 Remove this code from orig-tarball and modify import directives.
 .
 python-botocore (0.29.0+repack-1) UNRELEASED; urgency=medium
 .
   * Remove duplicated code from upstream tarball.
Author: TANIGUCHI Takaki <takaki@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: python-botocore-0.52.0/botocore/endpoint.py
===================================================================
--- python-botocore-0.52.0.orig/botocore/endpoint.py	2014-06-22 19:12:55.888338509 +0900
+++ python-botocore-0.52.0/botocore/endpoint.py	2014-06-22 19:12:55.884338571 +0900
@@ -17,8 +17,8 @@
 import time
 import threading
 
-from botocore.vendored.requests.sessions import Session
-from botocore.vendored.requests.utils import get_environ_proxies
+from requests.sessions import Session
+from requests.utils import get_environ_proxies
 import six
 
 import botocore.response
Index: python-botocore-0.52.0/botocore/retryhandler.py
===================================================================
--- python-botocore-0.52.0.orig/botocore/retryhandler.py	2014-06-22 19:12:55.888338509 +0900
+++ python-botocore-0.52.0/botocore/retryhandler.py	2014-06-22 19:12:55.884338571 +0900
@@ -17,8 +17,8 @@
 import logging
 from binascii import crc32
 
-from botocore.vendored.requests import ConnectionError
-from botocore.vendored.requests.packages.urllib3.exceptions import ClosedPoolError
+from requests import ConnectionError
+from urllib3.exceptions import ClosedPoolError
 
 from botocore.exceptions import ChecksumError
 
Index: python-botocore-0.52.0/botocore/awsrequest.py
===================================================================
--- python-botocore-0.52.0.orig/botocore/awsrequest.py	2014-06-22 19:12:55.888338509 +0900
+++ python-botocore-0.52.0/botocore/awsrequest.py	2014-06-22 19:13:52.571480844 +0900
@@ -17,14 +17,14 @@
 import inspect
 
 import six
-from botocore.vendored.requests import models
-from botocore.vendored.requests.sessions import REDIRECT_STATI
+from requests import models
+from requests.sessions import REDIRECT_STATI
 from botocore.compat import HTTPHeaders, file_type, HTTPResponse
 from botocore.exceptions import UnseekableStreamError
-from botocore.vendored.requests.packages.urllib3.connection import VerifiedHTTPSConnection
-from botocore.vendored.requests.packages.urllib3.connection import HTTPConnection
-from botocore.vendored.requests.packages.urllib3.connectionpool import HTTPConnectionPool
-from botocore.vendored.requests.packages.urllib3.connectionpool import HTTPSConnectionPool
+from urllib3.connection import VerifiedHTTPSConnection
+from urllib3.connection import HTTPConnection
+from urllib3.connectionpool import HTTPConnectionPool
+from urllib3.connectionpool import HTTPSConnectionPool
 
 
 logger = logging.getLogger(__name__)
Index: python-botocore-0.52.0/botocore/utils.py
===================================================================
--- python-botocore-0.52.0.orig/botocore/utils.py	2014-06-20 06:28:15.000000000 +0900
+++ python-botocore-0.52.0/botocore/utils.py	2014-06-22 19:15:44.709784035 +0900
@@ -15,7 +15,7 @@
 
 from .exceptions import InvalidExpressionError, ConfigNotFound
 from .compat import json
-from .vendored import requests
+import requests
 
 
 logger = logging.getLogger(__name__)
